internal/auth/dist/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Changelog
```

Comment:
This file seems to be unnecessary for the merge unless we want to introduce per-package changelogs.